src/main/java/cn/nukkit/item/Item.java

```diff
@@ -906,11 +906,15 @@ public int getMaxStackSize() {
     }
 
     final public Short getFuelTime() {
+        int id = this.id;
+        if (this instanceof ItemBlock) {
+            id = 255 - this.id;
+        }
```

Comment:
If the proposed change above gets implemented, I think there would be no need to do this because the id would never collide (note: I could be wrong, I couldn't test yet)